python/qibuild/project.py

```diff
@@ -177,6 +177,24 @@ def using_make(self):
     def verbose_make(self):
         return self.build_config.verbose_make
 
+    def write_project_cmake(self):
+        """ Write the project.cmake file. This will be read by
+        qibuild-config.cmake to set QI_SDK_VERSION.
+        """
+        to_write = """
+#############################################
+#QIBUILD AUTOGENERATED FILE. DO NOT EDIT.
+#############################################
+
+# Add qiproject version to CMake
+set(QI_SDK_VERSION %s CACHE STRING "" FORCE)
```

Comment:
I believe QI_SDK_ prefix is already used elsewhere.

Mabye use QI_PROJECT_VERSION instead ?

Reply:
Sounds good, I'll fix it.